src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -143,6 +143,7 @@ function WorkspaceCategoriesPage({policy, policyCategories, route}: WorkspaceCat
                 medium
                 onPress={navigateToCategoriesSettings}
                 icon={Expensicons.Gear}
+                iconStyles={[styles.mr2]}
```

Comment:
Is this change intended?

Reply:
Yes, icon on settings button was missing right margin on that screen. Double-checked in figma